client/src/components/ReviewTile.js

```diff
@@ -1,10 +1,22 @@
 import React from "react"
 
 const ReviewTile = (props) => {
+  let upVotes = 0
+  let downVotes = 0
+  
+  props.votes.forEach(vote => {
+    if(vote.direction === "up") {
+      upVotes++
+    } else {
+      downVotes++
+    }
+  })
```

Comment:
This is fine, you could also potentially compute this on the backend before returning the votes from the backend API first, so that we don't need to compute these values in React. however, you may better evaluate how you want to do this when adding the "create votes" feature.